includes/DefaultSettings.php

```diff
@@ -5335,6 +5335,7 @@
  * Unsetting core actions will probably cause things to complain loudly.
  */
 $wgActions = array(
+	'approvedraft'   => true,
```

Comment:
This shouldn't be here. Instead, please add to $wgActions in the code of our extensions, similarly to how we add items to $wgHooks and other globals.